patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -162,7 +162,20 @@
              }
          }
      }
-@@ -782,7 +832,7 @@
+@@ -457,7 +507,11 @@
+ 
+             try
+             {
+-                iblockstate.func_177230_c().func_176204_a(this, p_180496_1_, iblockstate, p_180496_2_);
++                net.minecraftforge.event.world.BlockEvent.NeighborNotifyEvent event = net.minecraftforge.event.ForgeEventFactory.onNeighborNotify(this, p_180496_1_, this.func_180495_p(p_180496_1_));
++                if(!event.isCanceled())
++                {
++                    iblockstate.func_177230_c().func_176204_a(this, p_180496_1_, iblockstate, p_180496_2_);
```

Comment:
Unindent this line, remove `{` and `}`, add a comment after the if statement for a minimal patch. You'll end up with a 2+ patch instead of 5+ 1-